dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/DependencyGraphBuilder.java

```diff
@@ -271,9 +236,12 @@ public static UnresolvableArtifactProblem createUnresolvableArtifactProblem(
    * @param firstNode node to start traversal
    */
   public static DependencyGraph levelOrder(DependencyNode firstNode) {
-
     DependencyGraph graph = new DependencyGraph();
 
+    return levelOrder(firstNode, graph);
+  }
+
+  private static DependencyGraph levelOrder(DependencyNode firstNode, DependencyGraph graph) {
```

Comment:
Thinking about other implementation of `DependencyGraph`, this `levelOrder` method of converting DependencyNode to `DependencyGraph` is specific to `DependencyNode`. It's because `graph.addPath(path);` in this method comes from the fact that current `DependencyGraph` consists of `DependnecyPath`s.

Do you think moving this method inside DependencyGraph, as `DependencyGraph.from(DependencyNode root)` is a good idea?

Reply:
Good idea. Done, and that simplified some more code too.